DependencyInjection/Compiler/DoctrineMongoDBMappingsPass.php

```diff
@@ -48,15 +48,15 @@ public function __construct($driver, array $namespaces, array $managerParameters
     }
 
     /**
-     * @param array    $mappings          Hashmap of directory path to namespace
-     * @param string[] $managerParameters List of parameters that could which object manager name
-     *                                    your bundle uses. This compiler pass will automatically
-     *                                    append the parameter name for the default entity manager
-     *                                    to this list.
-     * @param bool     $enabledParameter  Service container parameter that must be present to
-     *                                      enable the mapping. Set to false to not do any check,
-     *                                      optional.
-     * @param string[] $aliasMap          Map of alias to namespace.
+     * @param array        $mappings          Hashmap of directory path to namespace
+     * @param string[]     $managerParameters List of parameters that could which object manager name
+     *                                        your bundle uses. This compiler pass will automatically
+     *                                        append the parameter name for the default entity manager
+     *                                        to this list.
+     * @param string|false $enabledParameter  Service container parameter that must be present to
+     *                                 enable the mapping. Set to false to not do any check,
```

Comment:
Nitpicking but the PHPDoc is off few spaces for `$enabledParameter` here and in other classes :)